### PR TITLE
[Hub Generated] Move private branch 'dev-azuredata-preview' to public

### PR DESCRIPTION
**Pull request by mozehgir**
This PR moves the private branch that was created into public. The branch has already been reviewed, to view the status, please go to the [OpenAPI Hub](https://portal.azure-devex-tools.com/branch/Azure/azure-rest-api-specs-pr/dev-azuredata-preview...master/).

### CODE REVIEW

**Comment by rajmusuku** [comment]
These changes were already approved in private repo. Below is the private PR link.
https://github.com/Azure/azure-rest-api-specs-pr/pull/748

File: specification/azuredata/resource-manager/readme.csharp.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+```yaml $(csharp)
+csharp:
+  azure-arm: true
+  license-header: MICROSOFT_MIT_NO_VERSION
+  payload-flattening-threshold: 1
+  clear-output-folder: true
+  client-side-validation: false
+  namespace: Microsoft.AzureData
+  output-folder: $(csharp-sdks-folder)/azuredata/management/Microsoft.AzureData/Generated
```
Comment (erich-wang) [inline specification/azuredata/resource-manager/readme.csharp.md]: change to $(csharp-sdks-folder)/azuredata/Microsoft.Azure.Management.AzureData/src/Generated

File: specification/azuredata/resource-manager/readme.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+This section describes what SDK should be generated by the automatic system.
+This is not used by Autorest itself.
+
+```yaml $(swagger-to-sdk)
+swagger-to-sdk:
+  - repo: azure-sdk-for-python
+  - repo: azure-sdk-for-java
+  - repo: azure-sdk-for-go
+  - repo: azure-sdk-for-js
+  - repo: azure-sdk-for-ruby
```
Comment (erich-wang) [inline specification/azuredata/resource-manager/readme.md]: add one more line:
`- repo: azure-sdk-for-net`

**Review by erich-wang — CHANGES_REQUESTED** [review]

**Comment by mozehgir** [comment]
@rajmusuku - can you please address any issues with the content.

**Comment by rajmusuku** [comment]
@erich-wang Just to let you know, this RP is already in public preview and we released a portal extension as well. Swagger and the other stuff was reviewed earlier, but we forgot to merge it to Public repo since we don't use SDK part of this application.

File: specification/azuredata/resource-manager/readme.csharp.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+Please also specify `--csharp-sdks-folder=<path to "SDKs" directory of your azure-sdk-for-net clone>`.
+
+```yaml $(csharp)
+csharp:
+  azure-arm: true
+  license-header: MICROSOFT_MIT_NO_VERSION
+  payload-flattening-threshold: 1
+  clear-output-folder: true
+  client-side-validation: false
+  namespace: Microsoft.AzureData
```
[the remark this answers is not in the document]
Comment (rajmusuku) [reply specification/azuredata/resource-manager/readme.csharp.md]: Is this a standard norm that we need to include 'Management' word? RP has the namespace as Microsoft.AzureData

File: specification/azuredata/resource-manager/Microsoft.AzureData/preview/2017-03-01-preview/azuredata.json [hunk 4]
```diff
@@ -0,0 +1,873 @@
...
+        },
+        "version": {
+          "description": "Version of the Sql Server.",
+          "type": "string"
+        },
+        "edition": {
+          "description": "Sql Server Edition.",
+          "type": "string"
+        },
+        "registrationID": {
```
Comment (erich-wang) [inline specification/azuredata/resource-manager/Microsoft.AzureData/preview/2017-03-01-preview/azuredata.json]: should we use registration**Id**?

File: specification/azuredata/resource-manager/readme.csharp.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (erich-wang) [reply specification/azuredata/resource-manager/readme.csharp.md]: yes, it is standard for management sdk

File: specification/azuredata/resource-manager/Microsoft.AzureData/preview/2017-03-01-preview/azuredata.json (hunk 4 above)
Comment (rajmusuku) [reply specification/azuredata/resource-manager/Microsoft.AzureData/preview/2017-03-01-preview/azuredata.json]: this was already in public preview, may be we can leave it as is

File: specification/azuredata/resource-manager/readme.md (hunk 2 above)
Comment (rajmusuku) [reply specification/azuredata/resource-manager/readme.md]: Done

File: specification/azuredata/resource-manager/readme.md [hunk 5]
```diff
@@ -54,6 +54,7 @@ swagger-to-sdk:
   - repo: azure-sdk-for-go
   - repo: azure-sdk-for-js
   - repo: azure-sdk-for-ruby
+  - repo: azure-sdk-for-net
     after_scripts:
```
Comment (nschonni) [inline specification/azuredata/resource-manager/readme.md]: Think this is now applying to the net sdk instead of the Ruby one

**Review by erich-wang — APPROVED** [review]

File: specification/azuredata/resource-manager/readme.md (hunk 5 above)
Comment (erich-wang) [reply specification/azuredata/resource-manager/readme.md]: @rajmusuku , please move one line up for `- repo: azure-sdk-for-net`

**Review by erich-wang — APPROVED** [review]

**Pull request merged**